src/main/java/page/mentors/EditMentorsDetailsPage.java

```diff
@@ -0,0 +1,100 @@
+package page.mentors;
+
+import constants.Locators;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import page.base.Page;
+
+public class EditMentorsDetailsPage extends Page implements Locators.EditMentorsDetailsPage {
+    public EditMentorsDetailsPage(WebDriver driver) {
+        super(driver);
+    }
+    @FindBy (xpath = MENTORS_DETAILS_TAB_XPATH)
+    WebElement mentorsDetailsTab;
+    @FindBy (xpath =EDIT_MENTOR_TAB_XPATH)
+    WebElement editMentorsTab;
+    @FindBy (xpath =ARROW_BUTTON_XPATH)
+    WebElement arrowButton;
+    @FindBy (xpath =FIRSTNAME_INPUT_FIELD_XPATH)
+    WebElement firstNameInputField;
+    @FindBy (xpath =SECONDNAME_INPUT_FIELD_XPATH)
+    WebElement secondNameInputField;
+    @FindBy (xpath =EMAIL_INPUT_FIELD_XPATH)
+    WebElement emailInputField;
+    @FindBy (xpath =GROUPS_INPUT_FIELD_XPATH)
+    WebElement groupsInputField;
+    @FindBy (xpath =DELETE_GROUP_BUTTON_XPATH)
+    WebElement deleteGroupButton;
+    @FindBy (xpath =ADD_GROUP_BUTTON_XPATH)
+    WebElement addGroupButton;
+    @FindBy (xpath =COURSE_INPUT_FIELD_XPATH)
+    WebElement courseInputField;
+    @FindBy (xpath =DELETE_COURSE_BUTTON_XPATH)
+    WebElement deleteCourseButton;
+    @FindBy (xpath =ADD_COURSE_BUTTON_XPATH)
+    WebElement addCourseButton;
+    @FindBy (xpath =DISABLE_BUTTON_XPATH)
+    WebElement disableButton;
+    @FindBy (xpath =CLEAR_BUTTON_XPATH)
+    WebElement clearButton;
+    @FindBy (xpath =SAVE_BUTTON_XPATH)
+    WebElement saveButton;
+
+    public EditMentorsDetailsPage viewMentorsDetailTab(){
```

Comment:
As result of execution of this metod you'll be have MentorDetialsPage. So return type will be new MentorDetailsPage